imagedephi/redact/redact.py

```diff
@@ -66,8 +80,13 @@ def redact_images(
             redaction_plan.report_missing_rules()
         else:
             redaction_plan.execute_plan()
-            output_path = _get_output_path(image_file, output_dir)
-            redaction_plan.save(output_path, overwrite)
+            try:
+                output_path = _get_output_path(image_file, create_redact_dir(output_dir))
+                redaction_plan.save(output_path, overwrite)
```

Comment:
If possible, try to put only the lines which are expected to raise the exception inside the `try` block.